### ISSUE TITLE
qps_test failed

### ISSUE THREAD

**Issue by ctiller** [grpc/grpc#4001]
https://grpc-testing.appspot.com/job/gRPC_master/lastCompletedBuild/config=opt,language=c++,platform=macos/console

**Comment by dgquintas** [comment]
Still failing:

```
I1102 12:48:56.783644000 140735213904640 qps_test.cc:51] Running QPS test
D1102 12:48:56.784373000 140735213904640 test_config.c:155] test slowdown: machine=1.000000 build=1.000000 total=1.000000
E1102 12:48:56.785810000 140735213904640 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:48:56.787448000 140735213904640 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:48:56.789754000 140735213904640 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:48:56.790762000 140735213904640 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
D1102 12:48:56.793609000 4436484096 grpc_profiler.c:48] You do not have google-perftools installed, profiling is disabled [for qps_server.prof]
D1102 12:48:56.793631000 4436484096 grpc_profiler.c:52] To install on ubuntu: sudo apt-get install google-perftools libgoogle-perftools-dev
I1102 12:48:56.867981000 140735213904640 driver.cc:206] Warming up
I1102 12:49:16.869171000 140735213904640 driver.cc:212] Starting
E1102 12:50:16.795303000 140735213904640 driver.cc:226] assertion failed: server->stream->Read(&server_status)
```

**Comment by vjpai** [comment]
I believe this flake has been fixed by #4018 because the specific assertion failure is on the sync streaming server (stats collection). Will watch and see.

**Comment by vjpai** [comment]
I declare this closed based on a lack of failures in recent master builds since #4018 was merged.
